apps/src/gamelab/GameLab.js

```diff
@@ -235,20 +235,45 @@ GameLab.prototype.init = function (config) {
 GameLab.prototype.setupReduxSubscribers = function (store) {
   var state = {};
   var boundOnIsRunningChange = this.onIsRunningChange.bind(this);
+  var boundOnSelectedAnimationChange = this.onSelectedAnimationChange.bind(this);
```

Comment:
I wonder if there is any value in using the react pattern of rewriting all the instance methods to be prebound to the instance? i.e.
`this.onSelectedAnimationChange = this.onSelectedAnimationChange.bind(this);`
and then just referring to `this.onSelectedAnimationChange` everywhere instead of having these special `bound*` variables.